Makefile

```diff
@@ -0,0 +1,75 @@
+# --- Global -------------------------------------------------------------------
+O = out
+
+all: build test check-coverage lint  ## build, test, check coverage, lint
+	@if [ -e .git/rebase-merge ]; then git --no-pager log -1 --pretty='%h %s'; fi
+	@echo '$(COLOUR_GREEN)Success$(COLOUR_NORMAL)'
+
+clean:  ## Remove generated files
+	-rm -rf $(O)
+
+.PHONY: all clean
+
+# --- Build --------------------------------------------------------------------
+# Build all subdirs of ./cmd, excluding those with a leading underscore.
+BIN = $(O)/covid19
+
+build: | $(O)  ## Build binary
+	go build -o $(BIN) ./backend
+
+run: build  ## Run binary with testdata
+	$(BIN) < backend/testdata/data.csv
+
+.PHONY: build run
+
+# --- Test ---------------------------------------------------------------------
+COVERFILE = $(O)/coverage.txt
+COVERAGE = 73.8
+
+test: | $(O)  ## Run tests and generate a coverage file
+	go test -coverprofile=$(COVERFILE) ./...
+
+check-coverage: test  ## Check that test coverage meets the required level
+	@go tool cover -func=$(COVERFILE) | $(CHECK_COVERAGE) || $(FAIL_COVERAGE)
+
+cover: test  ## Show test coverage in your browser
+	go tool cover -html=$(COVERFILE)
+
+CHECK_COVERAGE = awk -F '[ \t%]+' '/^total:/ {print; if ($$3 < $(COVERAGE)) exit 1}'
+FAIL_COVERAGE = { echo '$(COLOUR_RED)FAIL - Coverage below $(COVERAGE)%$(COLOUR_NORMAL)'; exit 1; }
+
+.PHONY: test check-coverage cover
+
+# --- Lint ---------------------------------------------------------------------
+GOLINT_VERSION = 1.28.3
```

Comment:
there is a 1.30.0 version now. dont mind if you want to stay at 1.28.3 for reasons, just pointing it out.

Reply:
done.